### DISCUSSION TITLE
how do I get  infected file through to sandbox without download it to local machine?

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#833] [Q&A] [answered]
I glad for any help :) 
I cannot be authorized to RTR

thanks a lot
Nate

**Comment by jshcodes** [comment, marked as answer]
Hi @NSH531 -

Without RTR, you are pretty limited if you're not on the machine. A possible workaround could be to set the __Quarantine & Security Center Registration__ prevention policy setting to automatically upload to Falcon Sandbox (for either static or dynamic inspection).

**Comment by NSH531** [comment]
hi josh,
 I'll learn this topic


THANKS.
nate
